### PR TITLE
[mod_latestactions] namespaced

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
use namespaced classes

### Testing Instructions
code review
the module works as before

### CODE REVIEW

File: administrator/modules/mod_latestactions/tmpl/default.php [hunk 1]
```diff
@@ -19,16 +22,16 @@
...
+						<span class="icon-calendar" aria-hidden="true"></span> <?php echo HTMLHelper::_('date', $item->log_date, JText::_('DATE_FORMAT_LC5')); ?>
 					</div>
 				</div>
 			</div>
 		<?php endforeach; ?>
 	<?php else : ?>
 		<div class="row-fluid">
 			<div class="span12">
-				<div class="alert"><?php echo JText::_('MOD_LATEST_ACTIONS_NO_MATCHING_RESULTS');?></div>
+				<div class="alert"><?php echo Text::_('MOD_LATEST_ACTIONS_NO_MATCHING_RESULTS');?></div>
```
Comment (Quy) [inline administrator/modules/mod_latestactions/tmpl/default.php]:
```suggestion
				<div class="alert"><?php echo Text::_('MOD_LATEST_ACTIONS_NO_MATCHING_RESULTS'); ?></div>
```

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 6031ea6a7ff66c5d3ede10a0bc2cb81be1f56f2d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23342">issues.joomla.org/tracker/joomla-cms/23342</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 6031ea6a7ff66c5d3ede10a0bc2cb81be1f56f2d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23342">issues.joomla.org/tracker/joomla-cms/23342</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23342">issues.joomla.org/tracker/joomla-cms/23342</a>.</sub>

**Pull request merged**